sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -74,6 +74,40 @@ double joint_matmul(TOperand *A, TOperand *B, TResult *C, queue &q, int i) {
           auto m1 = it.get_local_id(0);
           auto n1 = it.get_local_id(1) / sgSize;
           auto sg = it.get_sub_group();
+#ifdef PREFETCH
+          size_t sgId = sg.get_group_id()[0];
+#ifdef VNNI
+          // each warp gets 8x32. But 16 warps are needed for each row (512)
+          size_t pm1 = sgId / 16;   // prefetch m1 (sgId/16)
+          size_t pn1 = sgId & 0x15; // prefetch n1 (sgId%16)
+#else
+          size_t pm1 = sgId / 8;   // prefetch m1 (sgId/8)
+          size_t pn1 = sgId & 0x7; // prefetch n1 (sgId%8)
+#endif
+          namespace syclex = sycl::ext::oneapi::experimental;
+          for (int p = 0; p < 3; p++)
+            joint_matrix_prefetch<8, 32>(
+                sg, A + (m2 * MCache2 + sgId * 8) * colsA + p * 32, colsA,
```

Comment:
here and everywhere below, please, do not use magic numbers, like 8, 32, etc....